contracts/03-tamagotchi-nft/tests/owning.rs

```diff
@@ -1,12 +1,69 @@
-use gtest::{Program, System};
-
-// TODO: 0️⃣ Copy tests from the previous lesson and push changes to the master branch
+use gstd::ActorId;
+use gtest::{Log, Program, System};
+use tamagotchi_nft_io::{TmgAction, TmgEvent};
 
 #[test]
 fn owning_test() {
```

Comment:
The tests also must be written for bad cases: Not owner is trying to Transfer, Approve or RevokeApproval